prpr/download.py

```diff
@@ -185,18 +186,22 @@ def _configure_firefox_driver(browser_settings, headless=False):
 
 
 def get_zip_urls(driver, revisor_url: str) -> list[str]:
-    logger.debug(f"Fetching from {revisor_url}...")
     with driver:
         return _get_zip_urls(driver, revisor_url)
 
 
 def _get_zip_urls(driver, revisor_url: str) -> list[str]:
+    logger.debug(f"Fetching from {revisor_url}...")
     page_load_timeout = PAGE_LOAD_TIMEOUT
     driver.maximize_window()
     driver.set_page_load_timeout(page_load_timeout)
     driver.get(revisor_url)
     try:
         element = driver.find_element_by_xpath(HISTORY_TAB_XPATH)
+        # The simple `element.click()` solution has stopped working on 19.05.2023.
```

Comment:
Let's make this comment shorter or even remove it:

`simple element.click() doesn't work here for some reason`